recipes/ligo-asimov/meta.yaml

```diff
@@ -0,0 +1,53 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Can you remove these comments at the top?

Reply:
Done.